src/System.Runtime.Serialization.Formatters/tests/BasicTests.cs

```diff
@@ -0,0 +1,27 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using Xunit;
+
+namespace System.Runtime.Serialization.Formatters.Tests
+{
+    public class BinaryFormatterTests 
+    {
+        [Fact]
+        public void Foo()
+        {
+            Assert.Equal(1, 1);
+        }
```

Comment:
I would like to keep it. I will be adding functional APIs and tests in a couple of days.